docs/monitor/view-active-alarms.md

```diff
@@ -9,17 +9,14 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/monitor/vie
 Every Netdata Agent comes with hundreds of pre-installed health alarms designed to notify you when an anomaly or
 performance issue affects your node or the applications it runs.
 
-As soon as you launch a Netdata Agent and [claim it](/docs/get/README.md#claim-your-node-on-netdata-cloud), you can view
-active alarms in both the local dashboard and Netdata Cloud.
+## Netdata Cloud
 
-## View active alarms in Netdata Cloud
-
-You can see active alarms from any node in your infrastructure in two ways: Click on the bell 🔔 icon in the top
-navigation, or click on the first column of any node's row in Nodes. This column's color changes based on the node's
-[health status](/health/REFERENCE.md#alarm-statuses): gray is `CLEAR`, yellow is `WARNING`, and red is `CRITICAL`.
+A War Room's [alerts indicator](/docs/cloud/war-rooms#indicators) displays the number of active `critical` (red) and
+`warning` (yellow) alerts for the nodes in this War Room. Click on either the critical or warning badges to open a
```

Comment:
@joelhans I am unsure if this is the right time to introduce the term alert. We need to change in Cloud, Agent GUI and documentation at the same time

Reply:
@manos-saratsis Good call. Changed this. I've gotten into the habit of writing alerts already.